src/components/utils/validation.js

```diff
@@ -1,97 +1,42 @@
 // @flow
 
+const charactersRegExpTest = (nameCandidate: string) => /^[a-zA-Z0-9_\-\\.]*$/.test(nameCandidate)
```

Comment:
@sgratch  Actually I'm not sure if we should throw away UTF chars. Check  I18NNameValidation which is used DiskAllocationModel - it [defines letters as all chars including UTF](https://github.com/oVirt/ovirt-engine/blob/e48484ccb3c1c8fcaed81023b234295a109ebd8b/frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/validation/I18NNameValidation.java#L11)